pkg/ucp/queue/client/client_test.go

```diff
@@ -50,7 +50,7 @@ func TestStartDequeuer(t *testing.T) {
 		Data:        []byte("{}"),
 	}, nil)
 
-	secondCall := mockCli.EXPECT().Dequeue(gomock.Any(), gomock.Any()).Return(nil, ErrInvalidMessage).After(firstCall)
+	secondCall := mockCli.EXPECT().Dequeue(gomock.Any(), gomock.Any()).Return(nil, ErrInvalidMessage).After(firstCall.Call)
```

Comment:
This is one of the changes that come with the mock updates, I guess.

Reply:
I think we are using this (second call should be after the first call test) only in this test.